_docs/principles/accessibility.md

```diff
@@ -198,61 +301,67 @@ Helix minimum spacing standards require: 
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+Pointer targets that are activated by a mouse, trackball, or stylus should be at least 44px x 44px. Large pointer targets can be helpful to users who have difficulty with fine motor skills.
 
-{% figure [caption:"Error state"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-inputs-2.svg"/>
-{% endfigure %}
+### Don't make users hover to find hidden controls
+
+Users that use screen readers or the keyboard to navigate to and select components rely on those components to be visible. Don't hide components as a way to reduce visual clutter. If a control cannot be seen, it cannot be verbally selected.
+
+### Write meaningful alternative text for images
+
+Screen readers read the text you enter in the `<alt>` tag that you associate with an image. If an image provides important contextual information, then use the `<alt>` tag to describe what is happening in the picture. If the image is decorative and does not provide important information, then add an empty `<alt>` tag. If you do not add an `<alt>` tag, some screen readers read the name of the image file, which introduces unnecessary confusion and results in a poor user experience.
 
 {% endcolumn %}
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+</div>
 
-#### Focus for Input Field
+</section>
 
-In this example, we provide a clear visual feedback that the focus has moved to the input field by changing the border color and adding a highlight to give the user clear indication of where they’re currently taking action.
+<section class="static-section"  markdown="1">
 
-{% endcolumn %}
+<div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-{% figure [caption:"Default state"][class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-inputs-3.svg"/>
-{% endfigure %}
+## Text guidelines
 
-{% endcolumn %}
+Consider the following text guidelines when you construct an accessible UI.
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+### Be succinct
 
-{% figure [caption:"Focused state"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-inputs-4.svg"/>
-{% endfigure %}
+Write short labels and be concise when writing helper text. Listening to lengthy field names and links can be cumbersome for users that use screen readers. The shorter the label, the more quickly a screen reader can work through the content.
+
+### Identify required fields
+
+Use the HTML5 `required` attribute to distinguish between required and optional fields. Screen readers cannot interpret an asterisk (``*``), which is the convention used to visually identify required fields.
```

Comment:
The term "field" is overloaded.

A "field" is (at minimum) a **label** (sometimes referred to as the "field name") and a **control** (sometimes referred to as the "input"). 

The `required` attribute should be used on the _control_ (e.g., `<input required>`, `<select required>`, and `<textarea required>`). 

The asterisk annotation should be applied to the _label_. As of HelixUI v0.16.0, the `.hxRequired` CSS class can be used to apply the asterisk without interfering with what is announced by screen readers.

Reply:
done.